pkg/detectors/groq/groq.go

```diff
@@ -0,0 +1,97 @@
+package groq
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	keyPat        = regexp.MustCompile(`\b(gsk_[a-zA-Z0-9]{52})\b`)
+)
+
+func (s Scanner) Keywords() []string {
+	return []string{"groq"}
```

Comment:
question: Do you think we could just use `gsk_` as the keyword?

Reply:
ya makes sense